R/Rules-class.R

```diff
@@ -139,13 +169,101 @@ validObject(.NextBestNCRM())
 ##' @keywords methods
 NextBestNCRM <- function(target,
                          overdose,
-                         maxOverdoseProb)
-{
-    .NextBestNCRM(target=target,
-                  overdose=overdose,
-                  maxOverdoseProb=maxOverdoseProb)
+                         maxOverdoseProb) {
+  .NextBestNCRM(
+    target = target,
+    overdose = overdose,
+    maxOverdoseProb = maxOverdoseProb
+  )
 }
 
+##' Next best dose based on NCRM rule and loss function
+##'
+##' This class is similar to NCRM class, the only difference is the addition of
+##' loss function. As in NCRM rule, first admissible doses are found, which are those
+##' with probability to fall in \code{overdose} category being below
```

Comment:
```suggestion
##' with probability to fall in `overdose` category being below
```
similarly below (in your class only) please use the markdown syntax instead of old Rd syntax

Reply:
Code updated